bot/discord/commands/server/create-donator.js

```diff
@@ -32,22 +32,25 @@ exports.run = async(client, message, args) => {
         message.channel.send("You are at your premium server limit")
         return;
     }
-
+    
+    const helpEmbed = new Discord.MessageEmbed()
+        .setColor("RED")
+        .setDescription(`List of servers: (use DBH!server create-donator <type> <name>)\n\n*Please note that some nodes might be having trouble connecting to the bot which may lead into this process giving out an error.*\n`)
```

Comment:
nodes dont have issues with connecting to the bot, but the bot may fail to connect to nodes